src/blocks/Info/Info.tsx

```diff
@@ -14,53 +13,58 @@ const b = block('info-block');
-                            <h2 className={b('content-title')}>{title}</h2>
-                            {buttons && (
-                                <div className={b('buttons')}>
-                                    {buttons.map((button, index) => (
-                                        <Button
-                                            key={index}
-                                            className={b('button')}
-                                            size="xl"
-                                            {...button}
-                                        />
-                                    ))}
-                                </div>
-                            )}
+                        <Col sizes={sizes} className={b('left')}>
+                            <Content
+                                title={title || leftContent?.title}
+                                text={leftContent?.text}
+                                links={leftContent?.links}
+                                theme={contentTheme}
+                                buttons={buttons || leftContent?.buttons}
+                                additionalInfo={leftContent?.additionalInfo}
+                                colSizes={{all: 12, md: 12}}
+                                className={b('content')}
+                                titleClassName={b('title')}
+                            />
                         </Col>
-                        <Col sizes={{lg: 4, sm: 6, all: 12}} offsets={{lg: 2, md: 0}}>
-                            <h2 className={b('sections-title')}>{sectionsTitle}</h2>
-                            <div className={b('links')}>
-                                {links &&
-                                    links.map((link, index) => (
-                                        <Link
-                                            {...link}
-                                            key={index}
-                                            className={b('link')}
-                                            colorTheme={textTheme}
-                                            theme={'normal'}
-                                            arrow={true}
-                                        />
-                                    ))}
-                            </div>
+                        <Col sizes={sizes} className={b('right')}>
+                            <Content
+                                title={sectionsTitle || rightContent?.title}
+                                text={rightContent?.text}
+                                links={
+                                    links?.map((link) => ({
+                                        ...link,
+                                        arrow: true,
+                                        theme: 'normal',
+                                    })) || rightContent?.links
+                                }
+                                theme={contentTheme}
```

Comment:
please put all the same props form this <Content> blocks to one variable

Reply:
fixed

Reply:
i mean same props of these two `<Content>` components: theme, additionalInfo, colSizes, etc.

Reply:
add commonProps with theme, className, colSizes. additionalInfo is different for two `<Content>`